src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -21,8 +22,7 @@ public String getTestedClassName() {
     @Test(dataProvider="testingData")
     public void testFileToFile(String fileIn, String extOut) throws Exception {
         String samFile= fileIn;
-        final File outFile = File.createTempFile(samFile + ".", extOut);
-        outFile.deleteOnExit();
+        final File outFile = BaseTest.createTempFile(samFile + ".", extOut);
```

Comment:
This method is inherited (here and in most of the test classes below), so can be just `createTempFile()`